packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -83,11 +85,19 @@ function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
     );
   }
 
-  const devices = parseIOSDevicesList(
-    child_process.execFileSync('xcrun', ['instruments', '-s'], {
-      encoding: 'utf8',
-    }),
-  );
+  let devices;
+  try {
+    devices = parseXctraceIOSDevicesList(
+      execa.sync('xcrun', ['xctrace', 'list', 'devices']).stderr,
```

Comment:
Why do we parse from stderr instead of stdout? I'm debugging an issue where no devices are returned, and I see that changing this line to: `execa.sync('xcrun', ['xctrace', 'list', 'devices']).stdout,` outputs the expected list of devices and simulators.